angr/knowledge_plugins/cfg/cfg_node.py

```diff
@@ -87,13 +90,13 @@ def __init__(
         __repr__.
         """
 
-        self.addr = addr
+        self.addr: int = addr
```

Comment:
Java uses non-int addrs

Reply:
I think we need an `AddressType = int | SootAddressDescriptor`.

Reply:
That or make cfg-related types generic over an address type

Reply:
Let's go with @ltfish 's suggestion for now and improve address abstraction later